cli/tests/testdata/jupyter/integration_test.ipynb

```diff
@@ -665,19 +667,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 21,
    "id": "8e93df23-06eb-414b-98d4-51fbebb53d1f",
    "metadata": {},
    "outputs": [
     {
-     "ename": "TypeError",
-     "evalue": "Cannot read properties of undefined (reading 'broadcast')",
-     "output_type": "error",
-     "traceback": [
-      "Stack trace:",
-      "TypeError: Cannot read properties of undefined (reading 'broadcast')",
-      "    at <anonymous>:2:20"
-     ]
+     "data": {
+      "text/html": [
+       "<b>Complete ✅</b>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {},
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
     }
```

Comment:
For a separate PR, we should make sure to not send an execute_result if we don't have any payload.

Reply:
Hm, that looks like a bug, we should be skipping this message if the only value is `undefined`.